code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -303,54 +348,170 @@
+
+/obj/item/bodypart/chest/alien
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "alien_chest_s"
+	dismemberable = 0
+	max_damage = 500
+	animal_origin = ALIEN_BODYPART
+
+/obj/item/bodypart/chest/devil
+	dismemberable = 0
+	max_damage = 5000
+	animal_origin = DEVIL_BODYPART
+
+/obj/item/bodypart/chest/larva
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "larva_chest_s"
+	dismemberable = 0
+	max_damage = 50
+	animal_origin = LARVA_BODYPART
+
 /obj/item/bodypart/l_arm
 	name = "left arm"
 	desc = "Did you know that the word 'sinister' stems originally from the \
 		Latin 'sinestra' (left hand), because the left hand was supposed to \
 		be possessed by the devil? This arm appears to be possessed by no \
 		one though."
+	icon_state = "default_human_l_arm"
 	attack_verb = list("slapped", "punched")
 	max_damage = 50
 	body_zone ="l_arm"
 	body_part = ARM_LEFT
 	px_x = -6
 	px_y = 0
 
+/obj/item/bodypart/l_arm/monkey
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "default_monkey_l_arm"
+	animal_origin = MONKEY_BODYPART
+	px_x = -5
+	px_y = -3
+
+/obj/item/bodypart/l_arm/alien
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "alien_l_arm_s"
+	px_x = 0
+	px_y = 0
+	dismemberable = 0
+	max_damage = 100
+	animal_origin = ALIEN_BODYPART
+
+/obj/item/bodypart/l_arm/devil
+	dismemberable = 0
+	max_damage = 5000
+	animal_origin = DEVIL_BODYPART
+
 /obj/item/bodypart/r_arm
 	name = "right arm"
 	desc = "Over 87% of humans are right handed. That figure is much lower \
 		among humans missing their right arm."
+	icon_state = "default_human_r_arm"
 	attack_verb = list("slapped", "punched")
 	max_damage = 50
 	body_zone = "r_arm"
 	body_part = ARM_RIGHT
 	px_x = 6
 	px_y = 0
 
+/obj/item/bodypart/r_arm/monkey
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "default_monkey_r_arm"
+	animal_origin = MONKEY_BODYPART
+	px_x = 5
+	px_y = -3
+
+/obj/item/bodypart/r_arm/alien
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "alien_r_arm_s"
+	px_x = 0
+	px_y = 0
+	dismemberable = 0
+	max_damage = 100
+	animal_origin = ALIEN_BODYPART
+
+/obj/item/bodypart/r_arm/devil
+	dismemberable = 0
+	max_damage = 5000
+	animal_origin = DEVIL_BODYPART
+
 /obj/item/bodypart/l_leg
 	name = "left leg"
 	desc = "Some athletes prefer to tie their left shoelaces first for good \
 		luck. In this instance, it probably would not have helped."
+	icon_state = "default_human_l_leg"
 	attack_verb = list("kicked", "stomped")
 	max_damage = 50
 	body_zone = "l_leg"
 	body_part = LEG_LEFT
 	px_x = -2
 	px_y = 12
 
+/obj/item/bodypart/l_leg/monkey
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "default_monkey_l_leg"
+	animal_origin = MONKEY_BODYPART
+	px_y = 4
+
+/obj/item/bodypart/l_leg/alien
+	icon = 'icons/mob/animal_parts.dmi'
+	icon_state = "alien_l_leg_s"
+	px_x = 0
+	px_y = 0
+	dismemberable = 0
+	max_damage = 100
+	animal_origin = ALIEN_BODYPART
+
+/obj/item/bodypart/l_leg/devil
+	dismemberable = 0
+	max_damage = 5000
+	animal_origin = DEVIL_BODYPART
```

Comment:
No sprite for this, because I don't care about devils, and they can never become gibbed anyway.